src/app/admin-app.component.ts

```diff
@@ -1,15 +1,25 @@
 import { Component, OnInit } from "@angular/core";
-import { Store } from "@ngrx/store";
-import { authActions } from "./auth/store/actions";
+import { MatDialog } from "@angular/material/dialog";
+import { AdminWarningComponent } from "./shared/components/admin-warning/admin-warning.component";
+import { PersistenceService } from "./shared/services/persistence.service";
 
 @Component({
   selector: "admin-app-root",
   templateUrl: "./admin-app.component.html",
 })
 export class AdminAppComponent implements OnInit {
-  constructor(private store: Store) {}
+  constructor(
+    private dialogService: MatDialog,
+    private persistenceService: PersistenceService
+  ) {}
 
   ngOnInit(): void {
-    this.store.dispatch(authActions.getCurrentUser());
+    const preference = this.persistenceService.get("doNotWarn");
```

Comment:
this variable is not required. We can directly write this as 
const doNotWarn = !!this.persistenceService.get("doNotWarn");;